io.github.lpub3d.LPub3D.json

```diff
@@ -0,0 +1,183 @@
+{
+  "id": "io.github.lpub3d.LPub3D",
```

Comment:
Shouldn't it be `io.github.trevorsandy.LPub3D`?

Reply:
I followed the flatpack guidance and recommendation (to my understanding) and chose to go with the application name versus the GitHub username. 

If the current configuration is contributing to the packaging failure (or will contribute to runtime failure), I will be pleased to change it.

Cheers,

Reply:
You don't need to redo the submission. Update the PR and changing the title works fine.

the guidance it to use a domain you own (or has been delegated). This would work if you owned `https://github.com/lpub3d` as an organization or username.